cli/types/cypress.d.ts

```diff
@@ -1252,14 +1252,14 @@ declare namespace Cypress {
      *
      * @see https://on.cypress.io/getcookie
      */
-    getCookie(name: string, options?: Partial<Loggable & Timeoutable>): Chainable<Cookie | null>
+    getCookie(name: string, options?: Partial<CookieOptions>): Chainable<Cookie | null>
 
     /**
      * Get all of the browser cookies.
      *
      * @see https://on.cypress.io/getcookies
      */
-    getCookies(options?: Partial<Loggable & Timeoutable>): Chainable<Cookie[]>
+    getCookies(options?: Partial<CookieOptions>): Chainable<Cookie[]>
```

Comment:
When domain is not specified, does this return cookies from all domains?

Reply:
No, current superdomain. Updated the comment in https://github.com/cypress-io/cypress/pull/24471/commits/648449c6c507392bcee2a43f5cfba3d3225307c0.